app/src/ingest_bem_pdfs.py

```diff
@@ -67,13 +68,20 @@ def _ingest_bem_pdfs(
             _add_embeddings(chunks)
             db_session.add_all(chunks)
 
-            _save_json(file_path, chunks)
+            if save_json:
+                # Note that chunks are being added to the DB before saving the JSON.
+                # Originally, we thought about reviewing the JSON manually before adding chunks to the DB.
+                _save_json(file_path, chunks)
```

Comment:
I would implement this functionality (being able to manually adjust the json) treating the .pdf.json as caches: by first checking around line ~61 for the presence of a file at`file_path + .json` -- if there is one, load `chunks = json.load(contents_of_saved_json` and go straight to `_add_embeddings(chunks) \ db_session.add_call(chunks)`. So the method would be: ingest the .pdf, then if you don't like how it was loaded, edit the .pdf.json that this produced and re-ingest the pdf (which will then load the stored .pdf.json)

Reply:
(in a separate PR -- just responding to your comment here)

Reply:
Sounds great!